apps/dashboard/src/pages/_document.tsx

```diff
@@ -0,0 +1 @@
+export { Document as default } from "@artsy-studio/next";
```

Comment:
Does `_app` or `_document` ever get "extended", or do additional file contents ever get added here? If not, can these things be combined in some way?

I could also imagine spinning up a new app via a cli command that, say, outputs a `README` file that `@artsy-studio/next` sees in the directory and instead handles these exports at the lib level so that this step is never needed in userland. 

Also, unrelated: we use the `@artsy` namespace for all of our other libs. Reasoning for breaking that convention here with `@artsy-studio`? Or is the `-studio` an extension of the `@artsy` namespace?

Reply:
@artsy-studio isn't an NPM package. It's just an alias to a typescript project reference. These packages are never published.

Reply:
That makes sense re project references 👍

Reply:
Yeah, I'm assuming that 99% of the time these will never change. Unfortunately they're required to be there because next.js is file system based. 

Can always add them to vscode's file hide though.